.dockerignore

```diff
@@ -1,3 +1,2 @@
-node_modules
-build
-bin
+/node_modules
```

Comment:
leftover?

Reply:
why? now we don't do anything besides copying binary into docker. So we don't need to send `node_modules` (it can be huge and takes time)

Reply:
he is maybe asking about the extra `/` ??